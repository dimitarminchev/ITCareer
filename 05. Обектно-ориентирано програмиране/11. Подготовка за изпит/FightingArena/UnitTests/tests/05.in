CreateHero:Warrior:Simo
CreateHero:Assassin:Gosho
CreateHero:Priest:Silviya
CreateHero:Warrior:Dimitar
CreateWeapon:Silviya:MagicWand:MagicStick:5:5:5
CreateWeapon:Dimitar:Sword:SomeSword:5:5:5
Fight:Silviya:Simo
Fight:Silviya:Simo
Fight:Silviya:Gosho
Fight:Silviya:Gosho
Fight:Dimitar:Simo
Fight:Dimitar:Simo
Fight:Dimitar:Gosho
Fight:Dimitar:Gosho
CloseArena