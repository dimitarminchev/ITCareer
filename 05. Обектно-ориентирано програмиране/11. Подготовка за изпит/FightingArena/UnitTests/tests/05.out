Warrior: Simo joined the Arena!
Assassin: Gosho joined the Arena!
Priest: Silviya joined the Arena!
Warrior: Dimitar joined the Arena!
Successfully armed hero Silviya with weapon MagicWand!
Successfully armed hero Dimitar with weapon Sword!
Winner in the battle between Silviya and Simo is Silviya with 93.33.
Winner in the battle between Silviya and Simo is Silviya with 93.33.
Winner in the battle between Silviya and Gosho is Silviya with 93.33.
Winner in the battle between Silviya and Gosho is Silviya with 93.33.
Winner in the battle between Dimitar and Simo is Dimitar with 40.00.
Winner in the battle between Dimitar and Simo is Dimitar with 40.00.
Winner in the battle between Dimitar and Gosho is Dimitar with 40.00.
Winner in the battle between Dimitar and Gosho is Dimitar with 40.00.
Heroes: 4
Priest: Silviya - Level: 1
Experience: 20
MagicWand: MagicStick
  *Strength: 5
  *Agility: 5
  *Intelligence: 25
Power: 93.33
Warrior: Dimitar - Level: 1
Experience: 20
Sword: SomeSword
  *Strength: 20
  *Agility: 5
  *Intelligence: 5
Power: 40.00
Assassin: Gosho - Level: 0
Experience: 0
Power: 0.00
Warrior: Simo - Level: 0
Experience: 0
Power: 0.00